### DISCUSSION TITLE
Can I do inline fractions?

### DISCUSSION THREAD

**Discussion by wilfonba** [sleepymalc/VSCode-LaTeX-Inkscape#33] [Q&A]
I want to type inline fractions using A/B, but whenever I type "/" the fraction snippet that rewrites it as \frac{A}{B} is applied. Is there a way to override this snippet in inline math mode?

**Comment by wilfonba** [comment]
Resolved with cmd + z. Doesn't always work first try, but works good enough.

**Comment by sleepymalc** [comment]
Actually you can just type `A / B`, i.e., inserting a space before you type `/`. This is what I do when I want to avoid triggering the snippet.

**Reply by wilfonba** [reply]
That's a much better solution. I guess I'll have to break some habits to get used to this faster setup. Thanks!

**Reply by sleepymalc** [reply]
Yes, there's some learning curve unfortunately.
